components/shared/view/BasicImage.vue

```diff
@@ -1,32 +1,41 @@
 <template>
-  <b-image
-    :src="src || placeholder"
-    :src-fallback="placeholder"
-    :alt="alt"
-    ratio="1by1"
-    :class="customClass"
-    :rounded="rounded"
-    data-cy="type-image"
-    @error="onImageError">
-    <template #placeholder>
-      <NeoSkeleton full-size no-margin :circle="rounded" />
-    </template>
-  </b-image>
+  <figure
+    class="b-image-wrapper image is-1by1"
```

Comment:
I guess we should not use the naming `b-image`

Reply:
this was used (herited from bulma?)